src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        for (int i = 0; i < 3; i++) {
+            Ball ball = new Lottery().getRandomBall();
```

Comment:
Don't create several instances of the same class if you can use only one instance for your purposes. In this case, you're creating a new instance of `Lottery` each time you iterate through the loop. You should create a single `Lottery` instance before the loop and use it to call `getRandomBall()`.